.expeditor/buildkite/cypress.sh

```diff
@@ -4,17 +4,15 @@ set -euo pipefail
 
 cd /workdir/e2e
 
-instances_to_test=( \
-    "a2-iamv2-local-fresh-install-${CHANNEL}.cd.chef.co" \
-    "a2-iamv2-local-inplace-upgrade-${CHANNEL}.cd.chef.co" \
-    "a2-iamv2p1-local-fresh-install-${CHANNEL}.cd.chef.co" \
-    "a2-iamv2p1-local-inplace-upgrade-${CHANNEL}.cd.chef.co" \
-    "a2-local-fresh-install-${CHANNEL}.cd.chef.co" \
-    "a2-perf-test-single-local-inplace-upgrade-${CHANNEL}.cd.chef.co" \
-)
+data=$(curl --silent https://a2-${CHANNEL}.cd.chef.co/assets/data.json)
+instances_to_test=$(jq -nr --arg-json data "$data" '$data[] | select(.tags[] | contains ("e2e")) | .fqdn')
 
 for instance in ${instances_to_test[*]}
 do
+  if !jq -enr --arg fqdn "$instance" --argjson data "$data" '$data[] | select(.fqdn == $fqdn) as $i | "saml" | IN($i.tags[])'; then
```

Comment:
I think `IN` requires at least jq 1.6, which doesn't appear available via hab, but I'm not sure what is available where this script runs.

Reply:
Oh, that's unfortunate. I'll see if there's anything to about that...

Reply:
Open PR is here: https://github.com/habitat-sh/core-plans/pull/2719

Although, another odd thing is that 1.5 is uploaded as version 1.10 so we might need to figure out if there is anything we can do there.

Reply:
✔️ jq code rewritten